gdal/doc/source/user/virtual_file_systems.rst

```diff
@@ -229,6 +229,8 @@ Starting with GDAL 2.2, the ``AWS_REQUEST_PAYER`` configuration option may be se
 
 The ``AWS_S3_ENDPOINT`` configuration option defaults to s3.amazonaws.com.
 
+The ``AWS_HTTPS`` configuration option defaults to ``YES``.
```

Comment:
```suggestion
The :decl_configoption:`AWS_HTTPS` configuration option defaults to ``YES``.
```
To automatically grow the list at https://gdal.org/user/configoptions.html#list-of-configuration-options-and-where-they-apply. Would be need to convert the neighbouring options too